dist/images/install.sh

```diff
@@ -1486,6 +1486,8 @@ spec:
                   type: boolean
                 enableLb:
                   type: boolean
+                enableEcmp:
```

Comment:
crd change should also update helm charts

Reply:
The second file is update for helm charts.
The charts in acp can be updated when upgrade kube-ovn image.